src/server/game/Spells/SpellInfoCorrections.cpp

```diff
@@ -3673,7 +3673,7 @@ void SpellMgr::LoadSpellInfoCorrections()
     });
 
     // Krolmir, Hammer of Storms (13010)
```

Comment:
need to remove the comment too.
```suggestion
    // Riding Jokkum
```